docs/01-start-here/05-development-tips.md

```diff
@@ -19,6 +19,19 @@ You can run the Jasmine unit test suite with
 make test
 ```
 
+### Things to do before you commit
```

Comment:
What about rather than `Things to do before you commit`, you just add a section on `Useful Make commands` and add the ones you found useful with the description?

`If you are wondering what other options make has, you can simply type `make` at the comment line.` +1 for keeping this line though.

Reply:
haha, there's always room for another PR, they are free!
although if I wanted it to be editable easily perhaps I should have put in the wiki or google doc somewhere?

Reply:
Docs in Github are usually best for discoverability. Thanks @johnduffell !